src/proto.c

```diff
@@ -97,13 +76,7 @@ int proto_start(const struct settings *settings)
 	 * TODO: later support dynamic protocol selection.
 	 */
 
-	proto = get_proto_ops(settings->proto);
-	if (proto == NULL)
-		return -EINVAL;
-
-	hal_log_info("proto_ops: %s", proto->name);
-
-	return proto->probe(settings->host, settings->port);
```

Comment:
This change doesn't seem to belong to this commit.

Reply:
It is related. I remove settings->proto/hoss/port from the settings source file.